reactDashboard/src/components/DataCharts/DataChartsUtils.test.js

```diff
@@ -12,28 +12,59 @@ import {
 } from "../DataCharts/DataChartsConsts";
 
 describe("commonChartConfiguration", () => {
-  const mockData = [datasetConfiguration("testLabel", undefined, "#767676")];
+  const mockData = [datasetConfiguration("testLabel", "#767676", undefined)];
```

Comment:
Why the reordering of parameters? the signature of the function is:
export function datasetConfiguration(datasetLabel, seriesData, colour, fillColour=null)